cli/src/commands/codegen.rs

```diff
@@ -19,11 +19,19 @@ pub struct Opts {
     /// Additional derives
     #[clap(long = "derive")]
     derives: Vec<String>,
+    /// Additional attributes
+    #[clap(long = "attribute")]
+    attributes: Vec<String>,
     /// Additional derives for a given type.
     ///
     /// Example `--derive-for-type my_module::my_type=serde::Serialize`.
     #[clap(long = "derive-for-type", value_parser = derive_for_type_parser)]
     derives_for_type: Vec<(String, String)>,
+    /// Additional attributes for a given type.
+    ///
+    /// Example `--attribute-type my_module::my_type=#[allow(clippy::all)]`.
+    #[clap(long = "attribute-for-type", value_parser = attribute_for_type_parser)]
+    attribute_for_type: Vec<(String, String)>,
```

Comment:
```suggestion
    attributes_for_type: Vec<(String, String)>,
```

Complete nit for consistency :)

Reply:
:heavy_check_mark: